setup/kubernetes/aws.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Set Up an Amazon Elastic Kubernetes Cluster
+description: Learn how to set up an Amazon EKS cluster for your Coder deployment.
+---
+
+This deployment guide shows you how to set up an Amazon Elastic Kubernetes
+Engine cluster on which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the [eksctl command line
+utility](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
+and the [AWS Command Line
+Interface](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+installed on your machine.
+
+## Step 1: Spin up a K8 Cluster
+
+The following will spin up a Kubernetes cluster using the eksctl command;
+replace the parameters and environment variables as needed to reflect those for
+your environment.
+
+```bash
+CLUSTER_NAME="MY_CLUSTER_NAME" \
+  SSH_KEY_PATH="~/.ssh/my-public-key.pub" REGION="us-east-2" \
+  eksctl create cluster \
+  --name "$CLUSTER_NAME" \
+  --version 1.17 \
+  --region "$REGION" \
+  --nodegroup-name standard-workers \
+  --node-type t3.medium \
```

Comment:
We may need to bump this up for a better experience. End users hopefully can scope their own node sizes, but a `t3.medium` is very limited (2 CPUs and 4 GB of RAM)

Reply:
@IAmHughes would be happy to bump this to large or xlarge -- what do you think of doing so now?

Reply:
I think at least `large` but more so a callout about sizing would be ideal.